ms2/src/org/labkey/ms2/proteinCoverageMap.jsp

```diff
@@ -32,4 +32,4 @@ LABKEY.requiresScript("util.js");
     Ext.QuickTips.init();
 </script>
 
-<%=bean.protein.getCoverageMap(bean.run, bean.showRunUrl, bean.aaRowWidth) %>
+<%=unsafe(bean.protein.getCoverageMap(bean.run, bean.showRunUrl, bean.aaRowWidth).toString())%>

```

Comment:
Actually... maybe... getCoverageMap() should return HtmlString. That pushes unsafe() closer to the rendering code. There are a few other usages that will (right now) need to toString() that HtmlString to render it, but it gets us closer to where we want to be.